eks-cluster.yaml

```diff
@@ -61,114 +57,92 @@ privateCluster:
 
 managedNodeGroups:
   - name: workspaces
-    desiredCapacity: 1
+    amiFamily: Ubuntu2004
+    spot: false
+    instanceTypes: ["m6i.xlarge", "m6i.2xlarge"]
+    desiredCapacity: 2
     minSize: 1
     maxSize: 10
-    # because of AWS addons
+    maxPodsPerNode: 110
     disableIMDSv1: false
-    # Please configure the size of the volume and additional features
-    # https://eksctl.io/usage/schema/#nodeGroups-volumeType
-    # https://aws.amazon.com/es/ebs/pricing/
     volumeSize: 300
     volumeType: gp3
     volumeIOPS: 6000
     volumeThroughput: 500
     ebsOptimized: true
-    # Use private subnets for nodes
-    # https://eksctl.io/usage/vpc-networking/#use-private-subnets-for-initial-nodegroup
     privateNetworking: true
-    ami: ami-009935ddbb32a7f3c
+    propagateASGTags: true
 
-    tags:
-      # EC2 tags required for cluster-autoscaler auto-discovery
-      k8s.io/cluster-autoscaler/enabled: "true"
-      k8s.io/cluster-autoscaler/gitpod: "owned"
     iam:
-      attachPolicyARNs: &attachPolicyARNs
+      attachPolicyARNs:
         - arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly
         - arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy
         - arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy
         - arn:aws:iam::aws:policy/ElasticLoadBalancingFullAccess
         - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
-      withAddonPolicies: &withAddonPolicies
+      withAddonPolicies:
         albIngress: true
         autoScaler: true
         cloudWatch: true
-        certManager: true
         ebs: true
-    # Using custom AMI images require the definition of overrideBootstrapCommand
-    # to ensure that nodes are able to join the cluster https://eksctl.io/usage/custom-ami-support/
-    overrideBootstrapCommand: |
-      #!/bin/bash
 
-      export CLUSTERNAME=gitpod
-      export NODEGROUP=workspaces
+    tags:
+      k8s.io/cluster-autoscaler/enabled: "true"
+      k8s.io/cluster-autoscaler/gitpod: "owned"
 
-      declare -a LABELS=(
-        eks.amazonaws.com/nodegroup="${NODEGROUP}"
-        gitpod.io/workload_workspace_services=true
-        gitpod.io/workload_workspace_regular=true
-        gitpod.io/workload_workspace_headless=true
-      )
+    labels:
+      gitpod.io/workload_workspace_regular: "true"
+      gitpod.io/workload_workspace_services: "true"
+      gitpod.io/workload_workspace_headless: "true"
 
-      export KUBELET_EXTRA_ARGS="$(printf -- "--max-pods=110 --node-labels=%s" $(IFS=$','; echo "${LABELS[*]}"))"
-      /etc/eks/bootstrap.sh ${CLUSTERNAME}
+    # using prebootstrap commands to let us insert containerd without
```

Comment:
Can you update the section at https://github.com/gitpod-io/gitpod-eks-guide/pull/53/files#diff-5e87efbc745f8ec6b225582500bc0d3a457fff5cbdf86e986be5fcf247033dc0L4, reflecting the changes after this?  🤔